addons/hr_holidays/static/src/owl/components/partner_im_status_icon/partner_im_status_icon.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.component.PartnerImStatusIcon" t-inherit-mode="extension">
+        <xpath expr="//span[hasclass('o_PartnerImStatusIcon')]" position="inside">
+            <t t-if="storeProps.partner.im_status === 'leave_online'">
+            	<i class="o_PartnerImStatusIcon_icon o-online fa fa-plane fa-stack-1x" title="Online" role="img" aria-label="User is online"/>
+            </t>
+            <t t-if="storeProps.partner.im_status === 'leave_offline'">
+            	<i class="o_PartnerImStatusIcon_icon o-away fa fa-plane fa-stack-1x" title="Out of office" role="img" aria-label="User is out of office"/>
```

Comment:
IMO modifier `o-away` is not exactly correct, because this icon happens when user is either away or offline while technically out of office. I agree the intent is to share colour of icon (orange), but from a logical POV this icon doesn't mean "away".

Reply:
I agree it's cleaner, but it is annoying... But ok created a new css rule and file just for that.